servers/dsub/jobs/__main__.py

```diff
@@ -64,6 +64,7 @@ def split_env_flag(name):
 app = connexion.App(__name__, specification_dir='./swagger/', swagger_ui=False)
 app.app.config['PROVIDER_TYPE'] = args.provider_type
 app.app.config['REQUIRES_AUTH'] = bool(args.requires_auth)
+app.app.config['LABEL_MIN_COUNT_FOR_RANK'] = 10
```

Comment:
I don't follow why this is no longer needed, but if it's not needed you should be able to revert this.